docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -198,7 +198,7 @@ Currently, you can only edit Snowflake Data Assets.
 
 ## Secure your GX API Data Source connection strings
 
-When you use the GX API and not GX Cloud to connect to Data Sources, you must obfuscate your sensitive Data Source credentials in your connection string. Data Source connection strings are persisted in [GX Cloud backend storage](/cloud/about_gx.md#gx-cloud-architecture). Connection strings containing plaintext credentials are stored as plaintext.
+When you use the GX API and not GX Cloud to connect to Data Sources, you can obfuscate sensitive Data Source credentials in your connection string as an additional security measure.
```

Comment:
I'm unclear about the use of the word "obfuscate" in this context.

To me obfuscate would mean:
- base64 encode the credentials before storing them
- hash/encrypt the credentials before storing them
- etc.
None of ^ work

Reply:
@wookasz "obfuscate" has been removed from the latest text version. For the record, obfuscate means: 

_render obscure, unclear, or unintelligible_

It did not have the meaning you attributed to it.